### PR TITLE
Richard

### PR DESCRIPTION
**Pull request by richardholst**
I have updated the install script to update the yaml config file with the download location for the currently logged on user

### CODE REVIEW

File: AutoPull_install.ps1 [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# make dir  
+mkdir "C:\Program Files (x86)\autopull\"
```
Comment (fredex42) [inline AutoPull_install.ps1]: will this error if the directory already exists?

**Review by fredex42 — CHANGES_REQUESTED** [review]
please can you include the existing powershell lines to register the protocol handler in the install script?

thanks a lot

File: AutoPull_install.ps1 (hunk 1 above)
Comment (richardholst) [reply AutoPull_install.ps1]: yes it will I will add a -force option so it does not error  for now ..then add  more eror checking later

**Review by fredex42 — APPROVED** [review]
:+1: 

Much appreciated!

**Pull request merged**